design/prototype-version-4/README.md

```diff
@@ -225,27 +207,24 @@ Below is a secondary heading, “Declaration”, before the text “By submittin
 
 This page includes a secondary heading “What happens next” followed by the content “We’ve sent you an email to confirm we have received your form.” This text is editable by the form creator within the admin side of the builder, meaning it can be customised as to the needs of the different forms or department and should match their internal service level agreements (SLAs).
 
-<br>
-
 ___
 
-<br>
-
 ## What we learned
 
-> Give a synopsis of the key findings from the round of research we tested this version.  
-> Include examples of found needs, pain points or quotes to help explain the findings. These do not have to be too detailed, as this should be covered in the research findings packs/decks - which can be linked to from here.
+This round of testing saw a reduction in issues with creating and editing a question.  
+
+We continued to see problems when form creators were previewing their form in a new tab, with users being confused where they were and also changing ‘hats’ from form creator to form filler.  
 
-<!-- REMOVE ME!!! after write up -->
-<!-- https://drive.google.com/drive/folders/1VOMG9a0XyV371_1LxV9qUaddkyuY_xjE -->
+### User research documentation
 
-<br>
+[Usability testing: Basic form building write up (GitHub)](../../research/2022-06-28_Basic_Form_Building.md)
```

Comment:
This link is a 404 - and I can't see what it should be linking to?

Reply:
I can't see how to edit this from this view, but the link should be https://github.com/alphagov/forms/blob/main/research/2022-06-28_Basic_Form_Building.md

Reply:
Ah yeah, it’ll go to where Oliver says, but it’ll only work when merged in because the filenames changed in Oliver’s recent rework. 
The link is relative to the current page/file location.